pkg/koordlet/resourceexecutor/executor.go

```diff
@@ -220,19 +243,25 @@ func (e *ResourceUpdateExecutorImpl) needUpdate(updater ResourceUpdater) bool {
 	return false
 }
 
-func (e *ResourceUpdateExecutorImpl) update(updater ResourceUpdater) error {
+func (e *ResourceUpdateExecutorImpl) update(updater ResourceUpdater, eventHelper *audit.EventHelper) error {
 	err := updater.update()
 	if err != nil {
 		klog.V(4).Infof("failed to update resource %s to %v, err: %v", updater.Key(), updater.Value(), err)
 		return err
 	}
 	klog.V(6).Infof("successfully update resource %s to %v", updater.Key(), updater.Value())
+
+	if eventHelper != nil && e.eventRecorder != nil {
+		executorMessage := fmt.Sprintf("successfully update resource: %s to value: %s", updater.Key(), updater.Value())
+		e.eventRecorder.Eventf(e.node, corev1.EventTypeNormal, eventHelper.Event.Reason, executorMessage)
```

Comment:
event is better to add on pod, besides, it seems that all callers of update function set the eventHelper as nil?

Reply:
+1

Reply:
@zwzhang0107 
the eventHelper is not nil , when only passed from the update function at /pkg/koordlet/runtimehooks/protocal/protocal.go , cause I think it's where we have the very first  set cgourps event when creating pod/container ?
please let me know if I got something wrong, thanks!